front/src/routes/forgot-password/index.js

```diff
@@ -1,12 +1,68 @@
 import { Component } from 'preact';
 import { connect } from 'unistore/preact';
+import get from 'get-value';
+
+import { validateEmail } from '../../utils/validateEmail';
+import { ForgotPasswordStatus, RequestStatus } from '../../utils/consts';
 import ForgotPasswordPage from './ForgotPasswordPage';
-import actions from '../../actions/forgotPassword';
 
 class ForgotPassword extends Component {
-  render(props, {}) {
-    return <ForgotPasswordPage {...props} />;
+  forgotPassword = async e => {
+    const { forgotPasswordEmail } = this.state;
+    if (e) {
+      e.preventDefault();
+    }
+
+    if (!validateEmail(forgotPasswordEmail)) {
```

Comment:
Forgot password crash here after changes

Reply:
Fixed, I changed
```javascript
import { validateEmail } from '../../utils/validateEmail';
```
to 
```javascript
import validateEmail from '../../utils/validateEmail';
```